src/app/services/transmart-services/transmart-resource.service.ts

```diff
@@ -369,19 +305,9 @@ export class TransmartResourceService {
    * @returns {Observable<blob>}
    */
   downloadExportJob(jobId: string) {
-    let endpoint = this.endpointService.getEndpoint();
-    let headers = new Headers();
-    headers.append('Content-Type', 'application/zip');
-    headers.append('Authorization', `Bearer ${endpoint.accessToken}`);
-    let url = `${endpoint.getUrl()}/export/${jobId}/download`;
-    const options = {
-      headers: new HttpHeaders({
-        'Content-Type': 'application/json',
-        'Authorization': `Bearer ${endpoint.accessToken}`
-      })
-    };
-    return this.http.get(url, {...options, responseType: 'blob'})
-      .catch(TransmartResourceService.handleError.bind(this));
+    let url = `${this.endpointUrl}/export/${jobId}/download`;
+    return this.http.get(url, {responseType: 'blob'})
```

Comment:
HTTP header "'Content-Type', 'application/zip'" is not added anymore, is it intentional?

Reply:
I wasn't aware of this change, but it seems right: the content of what we are sending to the backend is not `zip`, and that is what this header was telling.